README.md

```diff
@@ -1,3 +1,76 @@
-# alpaca-albatross-practicum-team1
+# Getting Started with Create React App
```

Comment:
This should be renamed to
`Getting Started with Volunteer app`

Reply:
Done